examples/quick_start.ipynb

```diff
@@ -0,0 +1,260 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "rsLXLb5wBeY2"
+      },
+      "outputs": [],
+      "source": [
+        "start_learning_rate = 1e-1\n",
+        "optimizer = optax.adam(start_learning_rate)\n",
+        "\n",
+        "# Initialize parameters of the model + optimizer.\n",
+        "params = jnp.array([0.0, 0.0])\n",
+        "opt_state = optimizer.init(params)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CpAvP1WSnsyM"
+      },
+      "source": [
+        "\n",
+        "\n",
+        "Next we write the update loop. The `GradientTransform` object contains an `update` function that takes in the current optimizer state and gradients and returns the `updates` that need to be applied to the parameters: `updates, new_opt_state = optimizer.update(grads, opt_state)`.\n",
+        "\n",
+        "Optax comes with a few simple [update rules](https://optax.readthedocs.io/en/latest/api.html#apply-updates) that apply the updates from the gradient transforms to the current parameters to return new ones: `new_params = optax.apply_updates(params, updates)`.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "TNkhz_nrB2lx"
+      },
+      "outputs": [],
+      "source": [
+        "# A simple update loop.\n",
+        "for _ in range(1000):\n",
+        "  grads = jax.grad(compute_loss)(params, xs, ys)\n",
+        "  updates, opt_state = optimizer.update(grads, opt_state)\n",
+        "  params = optax.apply_updates(params, updates)\n",
+        "\n",
+        "assert jnp.allclose(params, target_params), 'Optimization should result in target params'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XXEz3j7wPZUH"
+      },
+      "source": [
+        "### Custom optimizers\n",
+        "\n",
+        "Optax makes it easy to create custom optimizers by `chain`ing gradient transforms. For example, this creates an optimizer based on `adam`. Note that the scaling is `-learning_rate` which is an important detail since `apply_updates` is additive."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "KQNI2P3YEEgP"
+      },
+      "outputs": [],
+      "source": [
+        "# Exponential decay of the learning rate.\n",
+        "scheduler = optax.exponential_decay(\n",
+        "    -start_learning_rate,  # Note that we scale updates by -learning_rate.\n",
```

Comment:
Suggestion: add the arg name (https://optax.readthedocs.io/en/latest/api.html?highlight=optax.exponential_decay#optax.exponential_decay) since it may not be obvious

```suggestion
        "    init_value=-start_learning_rate,  # Note that we scale updates by -learning_rate.\n",
```